recipes/xeus-wren/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "xeus-wren" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "bed8835be599b1070eae44abf25f8a68f6fb4a0510e795975a25158928719dba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/xeus-wren/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host:
+    - cppzmq >=4.6.0,<5
+    - zeromq
+    - nlohmann_json >=3.2.0,<3.10
+    - wren >=0.4.0
+    - xeus >=2.0
+    - xtl >=0.7
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/xwren                                                           # [unix]
+    - test -f ${PREFIX}/share/jupyter/kernels/xwren/kernel.json                             # [unix]
+    - if not exist %LIBRARY_BIN%\\xwren.exe (exit 1)                                        # [win]
+    - if not exist %LIBRARY_PREFIX%\\share\\jupyter\\kernels\\xwren\\kernel.json (exit 1)   # [win]
+    - test -d ${PREFIX}/include/xeus-wren                                                   # [unix]
+    - test -f ${PREFIX}/include/xeus-wren/xinterpreter.hpp                                  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xeus-wren/xeus-wrenConfig.cmake                           # [unix]
+    - test -f ${PREFIX}/lib/cmake/xeus-wren/xeus-wrenConfigVersion.cmake                    # [unix]
+    - if not exist %LIBRARY_INC%\\xeus-wren\\xinterpreter.hpp (exit 1)                      # [win]
+    - if not exist %LIBRARY_LIB%\\cmake\\xeus-wren\\xeus-wrenConfig.cmake (exit 1)          # [win]
+    - if not exist %LIBRARY_LIB%\\cmake\\xeus-wren\\xeus-wrenConfigVersion.cmake (exit 1)   # [win]
+    # The kernel spec is installed in the general PREFIX
+    - if not exist %PREFIX%\\share\\jupyter\\kernels\\xwren\\kernel.json (exit 1)           # [win]
+
+about:
+  home: https://github.com/jupyter-xeus/xeus-wren
```

Comment:
This gives me a 404 error.

Reply:
was a copy paste error from a previous xeus kernel, sry